modules/features/profile/src/main/res/layout/view_user.xml

```diff
@@ -5,58 +5,71 @@
         android:layout_height="wrap_content"
-        android:layout_marginStart="10dp"
-        android:textAppearance="@style/C70"
-        android:textColor="?attr/primary_text_02"
-        app:layout_constraintBottom_toTopOf="@+id/lblUsername"
-        app:layout_constraintStart_toEndOf="@id/imgProfilePicture"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_chainStyle="packed"
-        tools:text="Not signed in" />
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/imgProfilePicture"
+        app:layout_constraintBottom_toBottomOf="@id/imgProfilePicture" />
 
     <TextView
         android:id="@+id/lblUsername"
-        android:layout_width="0dp"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginEnd="8dp"
-        android:textAppearance="@style/H40"
+        android:focusable="true"
+        android:gravity="center"
+        android:layout_marginTop="16dp"
+        android:textAppearance="@style/H20"
+        android:textAlignment="center"
         android:textColor="?attr/primary_text_01"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toStartOf="@+id/imgArrowRight"
-        app:layout_constraintStart_toStartOf="@id/lblSignInStatus"
-        app:layout_constraintTop_toBottomOf="@+id/lblSignInStatus"
-        tools:text="Setup Account" />
+        app:layout_constraintTop_toBottomOf="@id/subscriptionTierPill"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        tools:text="Kevin Jones" />
 
-    <ImageView
-        android:id="@+id/imgArrowRight"
-        android:layout_width="24dp"
-        android:layout_height="24dp"
-        android:layout_marginTop="8dp"
-        android:layout_marginEnd="20dp"
-        android:layout_marginBottom="8dp"
-        app:tint="?attr/primary_icon_02"
+    <TextView
+        android:id="@+id/lblUserEmail"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="16dp"
+        android:focusable="true"
+        android:gravity="center"
+        android:textAppearance="@style/H70"
+        android:textAlignment="center"
+        android:textColor="?attr/primary_text_01"
+        app:layout_constraintTop_toBottomOf="@id/lblUsername"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        tools:text="xxx@gmail.com" />
+
+    <Button
+        android:id="@+id/btnAccount"
+        android:text="@string/profile_set_up_account"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        style="@style/PCAccountOutlinedButton"
+        android:gravity="center"
+        android:layout_marginBottom="16dp"
```

Comment:
I think this is giving too much space below the button and above the row of stats because we also have a 24dp `marginTop` applied to all the items in the row of stats in `fragment_profile.xml`.